src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx

```diff
@@ -159,14 +292,19 @@ export const YearnDeposit = ({ api }: YearnDepositProps) => {
         return (
           <Approve
             asset={asset}
-            cryptoEstimatedGasFee=''
+            feeAsset={feeAsset}
+            cryptoEstimatedGasFee={bnOrZero(state.approve.estimatedGasCrypto)
+              .div(`1e+${feeAsset.precision}`)
+              .toFixed(5)}
             disableAction
-            fiatEstimatedGasFee=''
-            loading={false}
+            fiatEstimatedGasFee={bnOrZero(state.approve.estimatedGasCrypto)
+              .div(`1e+${feeAsset.precision}`)
+              .times(feeMarketData.price)
```

Comment:
Do either of these values need to be verified as valid numbers before doing the math?

Reply:
In this case I'm not too concerned since the `feeAsset` and `feeMarketData` is for ETH. We should always have a price and precision here.